packages/client/hmi-client/src/components/model/model-parts/tera-model-part-entry.vue

```diff
@@ -162,15 +164,27 @@ watch(
 	{ immediate: true }
 );
 
+watch(
+	() => props.unitExpression,
+	(newUnitExpression) => {
+		unitExpression.value = newUnitExpression;
+	}
+);
+
+watch(
+	() => props.name,
+	(newName) => {
+		nameText.value = newName;
+	}
+);
```

Comment:
Would `toRef` work? it may be a bit more succinct over using a watcher then you don't really have this idea of having 2 copies of the same thing.

```
const unitExpression = toRef(props. unitExpression)
...
```

Reply:
never mind, it works for enrichment but you can't modify it in the input field anymore. the toRef always points to the prop and isn't editable

Reply:
Oh I see - I thought we only need to watch for changes and do not need to edit